<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>412</width>
    <height>461</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dwarf Therapist</string>
  </property>
  <widget class="QWidget" name="main_widget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTableView" name="tbl_main">
      <property name="selectionMode">
       <enum>QAbstractItemView::SingleSelection</enum>
      </property>
      <property name="selectionBehavior">
       <enum>QAbstractItemView::SelectRows</enum>
      </property>
      <property name="sortingEnabled">
       <bool>false</bool>
      </property>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
      <attribute name="verticalHeaderVisible">
       <bool>false</bool>
      </attribute>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>412</width>
     <height>22</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu_File">
    <property name="title">
     <string>&amp;File</string>
    </property>
    <addaction name="act_connect_to_DF"/>
    <addaction name="act_read_dwarves"/>
    <addaction name="separator"/>
    <addaction name="act_exit"/>
   </widget>
   <widget class="QMenu" name="menu_Help">
    <property name="title">
     <string>&amp;Help</string>
    </property>
    <addaction name="act_about"/>
   </widget>
   <addaction name="menu_File"/>
   <addaction name="menu_Help"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="act_connect_to_DF">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/img/img/connect.png</normaloff>:/img/img/connect.png</iconset>
   </property>
   <property name="text">
    <string>Connect To DF</string>
   </property>
   <property name="toolTip">
    <string>Attemp connecting to a running copy of Dwarf Fortress</string>
   </property>
  </action>
  <action name="act_exit">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/img/img/door_out.png</normaloff>:/img/img/door_out.png</iconset>
   </property>
   <property name="text">
    <string>E&amp;xit</string>
   </property>
   <property name="toolTip">
    <string>Close Dwarf Therapist</string>
   </property>
  </action>
  <action name="act_about">
   <property name="icon">
    <iconset resource="../images.qrc">
     <normaloff>:/img/img/help.png</normaloff>:/img/img/help.png</iconset>
   </property>
   <property name="text">
    <string>&amp;About</string>
   </property>
   <property name="toolTip">
    <string>Exactly what you think...</string>
   </property>
  </action>
  <action name="act_read_dwarves">
   <property name="text">
    <string>Read Dwarves</string>
   </property>
   <property name="toolTip">
    <string>Read the stuff...</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="../images.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>act_connect_to_DF</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>connect_to_df()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>act_exit</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>act_read_dwarves</sender>
   <signal>activated()</signal>
   <receiver>MainWindow</receiver>
   <slot>read_dwarves()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>314</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>connect_to_df()</slot>
  <slot>read_dwarves()</slot>
 </slots>
</ui>
